research-software-directory/RSD-as-a-service
# 882 project page

# Pagination on project and organisation overview pages

Closes #882

Changes proposed in this pull request:
* Fix pagination error on project page. The url param `page` uses 1 as starting page (page=1) on all overview pages now.
* Use same bottom pagination layout (smaller) on project and organisation overview page as on the software page
* Apply similar approach to organisations overview page. The organisation page will use preferred number of items saved in the cookie (similar to software and project overview pages). When there are no items it will show "Nothing to show" message (as on other pages)
* Changed the pagination label from "Per page" to "Items"
* Move old organisation card to user profile (it is still used there).
* Clean up unused imports and functions    

How to test:
* `make start` to build app and generate random data
* navigate to project overview page. Use pagination at the top and at the bottom and confirm that it works. Also try search and filtering options
* navigate to organisation overview page. Use pagination at the top and at the bottom. Also try search.
*  change number of items on the page to 48. Then move to project overview page. This settings  should be applied.

PR Checklist:
